### PR TITLE
Quarantine CanLaunchPhotinoWebViewAndClickButton

### PR DESCRIPTION
**Pull request by halter73**
Update: See https://github.com/dotnet/aspnetcore/issues/50802

### CODE REVIEW

**Comment by SteveSandersonMS** [comment]
@halter73 Can you add the rest of the quarantine issue template here so we know, e.g., what the failing build was?

**Review by Eilon — APPROVED** [review]

**Comment by SteveSandersonMS** [comment]
Please don't merge until this PR is updated to explain its reasoning.

**Comment by Eilon** [comment]
Yeah I don't exactly understand when/how this failed. It's clearly a failure on the links but I don't recall this test failing once I finished working on it...

**Comment by Eilon** [comment]
> > It's clearly a failure on the links
> 
> What is the "links"? Can you link to a failing build?

@SteveSandersonMS there are links in here: https://github.com/dotnet/aspnetcore/issues/50802

But I don't understand what build those are from because the tests of course passed on my PR. Are these AzDO builds from a regular "nightly" build or something that runs independently of PRs?

File: src/Components/WebView/test/E2ETest/WebViewManagerE2ETests.cs [hunk 1]
```diff
@@ -18,6 +18,7 @@ public class WebViewManagerE2ETests
     [ConditionalFact]
     [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX,
         SkipReason = "On Helix/Ubuntu the native Photino assemblies can't be found, and on macOS it can't detect when the WebView is ready")]
+    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/50802")]
```
Comment (halter73) [inline src/Components/WebView/test/E2ETest/WebViewManagerE2ETests.cs:21]: I left a comment on the issue mentioning that this seems specific to the aspnetcore-helix-matrix Windows.Amd64.Server2022.Open queue. We could probably use `SkipOnHelix` instead. Would that be better @SteveSandersonMS?

```suggestion
    [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/50802", Queues = "Windows.Amd64.Server2022.Open")]
```

Comment (SteveSandersonMS) [reply src/Components/WebView/test/E2ETest/WebViewManagerE2ETests.cs:21]: > Would that be better @SteveSandersonMS?

I'm happy with whatever you decide. The only thing I was trying to push for here is tracking the reasoning behind the decision (all I saw earlier was a quarantining PR with no description indicating why the test needed to be quarantined). I think this PR thread now contains enough detail for future-us to remember what was going on, so all's good!

**Pull request merged**